app/models/foreman_openscap/cve.rb

```diff
@@ -5,6 +5,7 @@ class Cve < ApplicationRecord
     has_many :oval_policies, :through => :host_cves
 
     scoped_search :relation => :host_cves, :on => :oval_policy_id, :rename => :oval_policy_id, :complete_value => false
+    scoped_search :relation => :host_cves, :on => :host_id, :rename => :host_id, :complete_value => false
```

Comment:
`:only_explicit => true` would be useful to avoid searching in this value without specifying attribute name, if you fix that, please also add the same to the one above